api/handler.go

```diff
@@ -30,7 +30,7 @@ import (
 
 	"github.com/golang/glog"
 	"github.com/google/cadvisor/events"
-	httpMux "github.com/google/cadvisor/http/mux"
```

Comment:
Any pointers to this convention?

On Mon, Nov 2, 2015 at 9:55 AM, Jimmi Dyson <notifications@github.com>
wrote:

> In api/handler.go
> <https://github.com/google/cadvisor/pull/936#discussion_r43658177>:
>
> > @@ -30,7 +30,7 @@ import (
> >
> >  	"github.com/golang/glog"
> >  	"github.com/google/cadvisor/events"
> > -	httpMux "github.com/google/cadvisor/http/mux"
>
> Yeah golang conventions
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/cadvisor/pull/936/files#r43658177>.
>